frontend/src/interfaces/releasePlans.ts

```diff
@@ -24,5 +24,17 @@ export interface IReleasePlanTemplatePayload {
     id?: string;
     name: string;
     description: string;
-    milestones?: IReleasePlanMilestone[];
+    milestones?: IReleasePlanMilestonePayload[];
```

Comment:
Why is this a payload instead of just `IReleasePlanMilestone`?

Reply:
I'll have a look with add strategies :)